sklearn/model_selection/_split.py

```diff
@@ -1448,10 +1448,14 @@ class GroupShuffleSplit(ShuffleSplit):
 
     test_size : float, int, None, optional (default=None)
         If float, should be between 0.0 and 1.0 and represent the proportion
-        of the dataset to include in the test split. If int, represents the
-        absolute number of test groups. If None, the value is set to the
-        complement of the train size. If ``train_size`` is also None, it will
-        be set to 0.2.
+
```

Comment:
```suggestion
```

Reply:
(the suggestion is to remove the blank line, which is maybe slightly non-obvious)